Bugzilla/WebService/Bug.pm

```diff
@@ -429,6 +429,12 @@ sub _translate_comment {
   # Don't load comment tags unless enabled
   if (Bugzilla->params->{'comment_taggers_group'}) {
     $comment_hash->{tags} = [map { $self->type('string', $_) } @{$comment->tags}];
+    $comment_hash->{tag_urls} = {};
+    foreach my $tag (@{$comment->tags}) {
+      my $url = $comment->tag_url($tag);
+      $comment_hash->{tag_urls}->{$tag} = $url if $url;
+    }
+    $comment_hash->{comment_tag_html} = $self->_get_comment_tag_html($comment);
```

Comment:
`comment_tag_html` shouldn't be exposed via the public API as it's internal only to the Bugzilla UI.

Instead this should be a `BugModel` endpoint that returns just the html (similar to `bug_model/cc/\d`).

Reply:
> `comment_tag_html` shouldn't be exposed via the public API as it's internal only to the Bugzilla UI.
> 
> Instead this should be a `BugModel` endpoint that returns just the html (similar to `bug_model/cc/\d`).

This would require a second call. I originally added the html option to the core add/remove endpoints to be able to do it all in one call. I will see what kind of performance change this would make.

Reply:
I thought that `comment_tag_html` is the only field used from that response.

Reply:
> I thought that `comment_tag_html` is the only field used from that response.

the refreshTags(), which just retrieves the current tag list,  does only use comment_tag_html, but  when you add/remove a tag, the endpoint will also return the refreshed comment_tag_html which is then passed to renderTags() to use. If I move the comment html potion to its own endpoint, I will need to first add/remove the tag with the first call, then call refreshTags() which will make a second call to get the rendered html. It might be just fine to do it that way and not noticeably slower. I will give it a try.

Reply:
Rather than making two calls you could return the html and the other fields in the internal endpoint.